core/common/src/main/java/alluxio/AbstractMasterClient.java

```diff
@@ -55,7 +51,15 @@ public AbstractMasterClient(MasterClientContext clientConf, InetSocketAddress ad
       Supplier<RetryPolicy> retryPolicySupplier) {
     super(clientConf, address, retryPolicySupplier);
     mMasterInquireClient = clientConf.getMasterInquireClient();
-    mConfMasterInquireClient = clientConf.getConfMasterInquireClient();
+  }
+
+  @Override
+  protected void beforeConnect()
```

Comment:
See comment above: I believe we can move these implementations back to `AbstractClient` ...